python/ray/_private/utils.py

```diff
@@ -298,6 +298,33 @@ def get_cuda_visible_devices():
 last_set_gpu_ids = None
 
 
+def get_xpu_visible_devices():
+    """Get the devices IDs in the XPU_VISIBLE_DEVICES environment variable.
+    Returns:
+        devices (List[str]): If XPU_VISIBLE_DEVICES is set, return a
+            list of strings representing the IDs of the visible XPUs.
+            If it is not set, returns empty list.
+    """
+    xpu_ids_str = os.environ.get("XPU_VISIBLE_DEVICES", None)
+    if xpu_ids_str is None:
+        return None
+
+    if xpu_ids_str == "":
+        return []
+
+    return list(xpu_ids_str.split(","))
+
+
+def get_current_accelerator():
+    return os.environ.get("RAY_ACCELERATOR", ray_constants.RAY_ACCELERATOR_DEFAULT)
```

Comment:
I'm concerned that this is using a top-level namespace. To users it can look like `RAY_ACCELERATOR` is a supported API, when really it's very experimental/alpha.

Reply:
I agree with your point of view, I think it is not a good design to use environment variables, because it is implicit, scattered configuration environment, resulting in uncontrollable.
But Ray still has a lot of environment variables in use, so I also added this environment variable.
For adding a top-level config API, how about add a argument in `ray.init` API, usage like this: `ray.init(acclerator="XPU")`

Reply:
>I agree with your point of view, I think it is not a good design to use environment variables, because it is implicit, scattered configuration environment, resulting in uncontrollable.
But Ray still has a lot of environment variables in use, so I also added this environment variable.

I would be more comfortable with `RAY_EXPERIMENTAL_ACCELERATOR`.

> For adding a top-level config API, how about add a argument in ray.init API, usage like this: ray.init(acclerator="XPU")

We should think about this carefully as there's big tradeoffs in how we expose Ray functionality at the API level. We'll need to propose this as an API change and list alternatives, their tradeoffs, and why this one is best.

Reply:
Changed `RAY_ACCELERATOR` to `RAY_EXPERIMENTAL_ACCELERATOR_TYPE`